content/community/index.page.tsx

```diff
@@ -1,528 +1,648 @@
+            height="102px"
+            src={require("../static/img/community/community-one.png")}
+          />
+          <img
+            alt="Person speaking at a conference"
+            className="parallax two"
+            src={require("../static/img/community/community-two.png")}
+          />
+          <img
+            alt="Person sitting and speaking"
+            className="parallax three"
+            src={require("../static/img/community/community-three.png")}
+          />
+          <img
+            alt="People chatting"
+            className="parallax four"
+            width="120px"
+            height="160px"
+            src={require("../static/img/community/community-four.png")}
+          />
+          <img
+            alt="Person speaking at Apex"
+            className="parallax five"
+            src={require("../static/img/community/community-five.png")}
+          />
+        </div>
+
+        <div className="mx-auto text-left col-lg-6 text-md-center hero-title">
+          <div className="d-flex flex-column-reverse align-items-center sm-align-items-start">
+            <img
+              src={require("../static/img/icons/arrow-down.svg")}
+              className="bounce-arrow"
+              alt="Down Arrow"
+            />
+            <h1 className="mb-0 main-title">
+              {translate("A Global Blockchain")}
+              <br className="until-sm" />
+              {translate("Community of")}
+              <span className="builders-wrap">Builders</span>
+              <br className="until-sm" />
+              {translate("and Innovators")}
+            </h1>
+            <h6 className="mb-3 eyebrow">{translate("XRPL Community")}</h6>
           </div>
-          <div className="row row-cols-2 row-cols-lg-4 card-deck">
-            {platforms.map((plat) => (
-              <a className="card mb-10" href={plat.link} target="_blank">
-                <div className="card-body">
-                  <div className="circled-logo">
-                    <img
-                      id={`platform-${plat.id}`}
-                      alt="(logo)"
-                      className={plat?.imgclasses || ""}
-                    />
-                  </div>
-                  <h4 className="card-title h5">{plat.name}</h4>
-                </div>
-                <div className="card-footer">&nbsp;</div>
+        </div>
+      </section>
+      {/* Community Table Section */}
+      <section id="community-table" className="hot-topics">
+        <h6 className="eyebrow-convo">{translate("Join the Conversation")}</h6>
+        <h4>{translate("Hot Topics Happening Now")}</h4>
+        <table>
+          <tr>
+            <td className="td-img">
+              <img className="discord-icon" alt="discord icon" />
+            </td>
+            <td>
+              {translate(
+                "AMA with Edge Wallet: Learn more about Edge Wallet and how they are building on the XRP Ledger."
+              )}
+            </td>
```

Comment:
This is not a new problem, so it doesn't need to be addressed here, but it seems like terrible UX for the description of the conversation not to be a link to the topic, so users have to click the (fairly small) arrow instead of clicking the topic itself.

Reply:
Lets make an issue for this and assign it to me please.

Reply:
Created #2333